src/dicionario.php

```diff
@@ -1,63 +1,63 @@
 <?php
```

Comment:
Eu to em dúvida se esse arquivo funciona bem quando importarmos o package, tinha de testar também.

Reply:
@leomoty O Travis não garante este?